types/react-elemental/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove rules from here and fix errors instead. New packages cannot have anything in their lint file except extends rule.